src/assets/styles/overrides.scss

```diff
@@ -21,6 +21,15 @@
   transform: translateY(-8px) scale(0.75);
 }
 
+// V-Cards
+.v-card {
+  .v-card__text {
+    font-weight: 300;
```

Comment:
Typo? In base.scss we only declare two font weights (400 and 700).

(And personally, I'd prefer to see "regular" and "bold". I shudder at the thought of different bold levels :P )

Reply:
oo yea, this was a mistake. Will fix!

Reply:
That being said, this is redundant. Removed!